src/web/templates/admin/accounts/tab.html

```diff
@@ -2,36 +2,65 @@
 
 {% block top_left_nav %}
     {{ admin_macros.admin_title(_('Accounts'), 'accounts') }}
```

Comment:
{{ admin_macros.admin_title(_('Accounts'), 'accounts') }}
```suggestion
    {{ admin_macros.admin_title(_('Staff'), 'staff') }}
```